packages/metrics_center/CHANGELOG.md

```diff
@@ -1,41 +1,47 @@
-# 1.0.1
+## NEXT
+
+- Fix `unnecessary_import` lint errors.
+- Update version titles in CHANGELOG.md so plugins tooling understands them.
```

Comment:
I'm not sure CHANGELOG needs to describe CHANGELOG changes; that's a bit meta :)